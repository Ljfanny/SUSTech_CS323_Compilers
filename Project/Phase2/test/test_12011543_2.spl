int test_12011543_2()
{
  int a,b;
  int x[12][01][15][43], y[120][115][43];
  while(0) {
    if(a) {
      int c = 12011543;
      x[0] = a;
    } 
    else if (b) {
      int d = a;
      y[0][0] = b;
    }
    else {
      float c = 1.2011543;
    }
  }
  return 0;
}