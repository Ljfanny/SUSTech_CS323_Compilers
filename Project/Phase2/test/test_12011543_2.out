Error type 15 at Line 6: redefine the same structure type: Apple
Error type 7 at Line 16: unmatching operands: PLUS
Error type 5 at Line 16: unmatching types on both sides of assignment
Error type 5 at Line 20: unmatching types on both sides of assignment
Error type 6 at Line 21: left side in assignment is rvalue
Error type 14 at Line 22: no such member: apple
Error type 5 at Line 22: unmatching types on both sides of assignment
Error type 5 at Line 26: unmatching types on both sides of assignment
Error type 8 at Line 30: incompatiable return type
Error type 4 at Line 36: redefine function: test_12011543_2
